src/templates/blogTemplate.js

```diff
@@ -0,0 +1,38 @@
+import Layout from 'components/layout'
+import SEO from 'components/seo'
+import { graphql } from 'gatsby'
+import React from 'react'
+
+const Template = ({ data }) => {
+  const { markdownRemark } = data
+  const { frontmatter, html } = markdownRemark
+  return (
+    <Layout>
+      <SEO title='Posts' />
```

Comment:
```suggestion
      <SEO title={frontmatter.title} />
```

Please use `frontmatter.title` as the page title in the SEO component.